mycore-ocfl/src/main/java/org/mycore/ocfl/user/MCROCFLXMLUserManager.java

```diff
@@ -0,0 +1,224 @@
+/*
+* This file is part of ***  M y C o R e  ***
+* See http://www.mycore.de/ for details.
+*
+* MyCoRe is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* MyCoRe is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.mycore.ocfl.user;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Optional;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jdom2.Document;
+import org.jdom2.JDOMException;
+import org.mycore.common.MCRPersistenceException;
+import org.mycore.common.MCRSystemUserInformation;
+import org.mycore.common.MCRUsageException;
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.common.content.MCRJDOMContent;
+import org.mycore.common.content.MCRStreamContent;
+import org.mycore.ocfl.MCROCFLObjectIDPrefixHelper;
+import org.mycore.ocfl.MCROCFLRepositoryProvider;
+import org.mycore.user2.MCRTransientUser;
+import org.mycore.user2.MCRUser;
+import org.mycore.user2.MCRUserManager;
+import org.mycore.user2.utils.MCRUserTransformer;
+import org.xml.sax.SAXException;
+
+import edu.wisc.library.ocfl.api.OcflOption;
+import edu.wisc.library.ocfl.api.OcflRepository;
+import edu.wisc.library.ocfl.api.exception.OverwriteException;
+import edu.wisc.library.ocfl.api.model.ObjectVersionId;
+import edu.wisc.library.ocfl.api.model.VersionInfo;
+
+/**
+ * XML Manager to handle MCRUsers in a MyCoRe OCFL Repository
+ * @author Tobias Lenhardt [Hammer1279]
+ */
+public class MCROCFLXMLUserManager {
+
+    private static final String THE_USER = "The User '";
```

Comment:
I would not extract a part of a message into a variable, even if code quality tools would complain about redundant strings here.